projects/storefrontapp/src/app/app.module.ts

```diff
@@ -67,6 +68,21 @@ if (!environment.production) {
         anonymousConsents: true,
       },
     }),
+    CdsModule.forRoot({
```

Comment:
Please revert this file.
We introduced a new shell app for cds here: https://sap-cx.slack.com/archives/CNFC4SCTZ/p1576519428004800